Helper/PaymentMethods.php

```diff
@@ -252,15 +252,32 @@ protected function fetchAlternativeMethods($country)
         return $paymentMethods;
     }
 
-    /**
-     * @return bool|int
-     */
-    protected function getCurrentPaymentAmount()
+    protected function getCurrentPaymentAmount(): float
     {
-        if (($grandTotal = $this->getQuote()->getGrandTotal()) > 0) {
+        $grandTotal = $this->getQuote()->getGrandTotal();
+
+        if (!\is_numeric($grandTotal)) {
```

Comment:
I think there's no need to specify the global namespace for these functions?
```suggestion
        if (!is_numeric($grandTotal)) {
```